werkItResources/workouts.txt

```diff
@@ -0,0 +1,4 @@
+push up | 10
+sit up | 10
+pull up | 10
+push up | 15
```

Comment:
Perhaps ignore the resource folder?

Reply:
okay, will take note of this for next commit. Thanks!